src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj

```diff
@@ -5,6 +5,14 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Paul asked for these to be removed in a previous PR of mine, they're obviously autogenerated but not sure if they cause any issues

Reply:
I tried the solution of leaving this in as it seemed to work in EI Functions, it worked for a while but it caused us problems later on when we merged more branches into ours. Started to generate and exclude feature.cs files. 

I really am annoyed Specflow is doing this to us.